movieland-persistence-jdbc/src/main/java/com/gromoks/movieland/dao/config/QueryConfig.java

```diff
@@ -63,5 +63,12 @@ public String getMoviesByIdSQL() {
                 " FROM movie m" +
                 " WHERE m.id = ?";
     }
+
+    @Bean
+    public String getUserByEmailSQL() {
+        return " SELECT id, nickname, email, password" +
```

Comment:
we don't need password in user details (select statement), but in where clause we need it